vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss

```diff
@@ -2,16 +2,25 @@
   background-color: $grey;
   color: $white;
   padding: 13rem 1rem 6rem 1rem;
-  background-image: asset_url('bg-location-2.webp');
+  background-image: asset_url("bg-location-2.webp");
+  @supports (background-image: asset_url("bg-location-2.webp")) {
```

Comment:
i think this logic might be backwards. if the `@supports` is `true`, shouldn't the `.webp` image be nested here?